charts/kas/Chart.yaml

```
@@ -5,5 +5,5 @@ description: TDF key access control service
 type: application
 maintainers:
   - name: Virtru
```

```suggestion
maintainers:
  - name: OpenTDF
    email: opentdf-dev@opentdf.com (or whatever)
```

would be the most useful thing, I think.

---

We don't have any MX records set up for opentdf.com yet, but that is a good idea. Maybe we can file a ticket with Doug Nerad to get that done?

---

Ah, I just looked and I think the support@opentdf.io email is indeed set up. Maybe we can merge this for now and I'll make a ticket to update with the correct email everywhere (including here).